coqpyt/tests/proof_file/test_proof_file.py

```diff
@@ -156,26 +158,6 @@ def test_nested_proofs(self):
         for i, step in enumerate(proofs[1].steps):
             assert step.text == steps[i]
 
-        # Close proofs
-        proof_file.exec(-1)
-        proof_file.add_step(proof_file.steps_taken - 1, "\nQed.")
-        proof_file.add_step(proof_file.steps_taken - 1, "\nQed.")
-        assert len(proof_file.proofs) == 2
-        assert len(proof_file.open_proofs) == 2
-
-        proof_file.exec(2)
-        assert len(proof_file.proofs) == 4
-        assert len(proof_file.open_proofs) == 0
-
-        # Re-open proofs
-        proof_file.exec(-2)
-        assert len(proof_file.proofs) == 2
-        assert len(proof_file.open_proofs) == 2
-
-        proof_file.exec(2)
-        assert len(proof_file.proofs) == 4
-        assert len(proof_file.open_proofs) == 0
```

Comment:
Why did you remove this?

Reply:
It was extracted to a new class `TestProofChangeNestedProofs` in `test_changes.py`. That was the only test in `test_proof_file.py` with `add_step`/`delete_step`/`change_steps`, which should only be tested in `test_changes.py`.